### ISSUE TITLE
Feat: Explore functionality/user story

### ISSUE THREAD

**Issue by lucarioams** [windingtree/videre#3]
https://github.com/windingtree/videre/blob/8335675554ebf506eaeb025b543369c71ac32901/packages/docs/docs/protocol/overview.md?plain=1#L35

What about the possibility to explore a large array of services without necessarily giving any parameters? [(Airbnb example)](https://www.airbnb.com/s/homes?search_mode=flex_destinations_search&date_picker_type=flexible_dates)

**Comment by mfw78** [comment]
This kind of feature would have to be modelled in the future. It may be possible in the future to look at this from a "curation" basis, whereby people are responsible for curating a list of places, and then this can be browsed as a list. 

I would allocate this a Low priority at the moment, and likely place this in a "Meta" category for features (ie. non-core, built on top of the data that's already around, or aggregating in some mehod).

**Comment by lucarioams** [comment]
agree, I think the latest Airbnb summer release is quite powerful with the categories being more prominent, they are definitely building a tool for remote workers and digital nomads, which will be a major pie of travellers in the future. So maybe we could have community curated categories, but indeed, this can remain as "non-core".
